docs/runtime/dotnet-metrics.md

```diff
@@ -0,0 +1,894 @@
+<!--- Hugo front matter used to generate the website version of this page:
+linkTitle: .NET
+--->
+
+# Semantic Conventions for .NET (CLR) Metrics
+
+**Status**: [Experimental][DocumentStatus]
+
+This document describes semantic conventions for .NET CLR runtime metrics in OpenTelemetry.
+
+<!-- Re-generate TOC with `markdown-toc --no-first-h1 -i` -->
+
+<!-- toc -->
+
+- [CLR Garbage Collection](#clr-garbage-collection)
+  - [Metric: `dotnet.gc.collections`](#metric-dotnetgccollections)
+  - [Metric: `dotnet.gc.objects.size`](#metric-dotnetgcobjectssize)
+  - [Metric: `dotnet.gc.memory.total_allocated`](#metric-dotnetgcmemorytotal_allocated)
+  - [Metric: `dotnet.gc.memory.committed`](#metric-dotnetgcmemorycommitted)
+  - [Metric: `dotnet.gc.heap.size`](#metric-dotnetgcheapsize)
+  - [Metric: `dotnet.gc.heap.fragmentation`](#metric-dotnetgcheapfragmentation)
+  - [Metric: `dotnet.gc.pause.time`](#metric-dotnetgcpausetime)
+- [Just-In-Time (JIT) Compiler](#just-in-time-jit-compiler)
+  - [Metric: `dotnet.jit.compiled_il.size`](#metric-dotnetjitcompiled_ilsize)
+  - [Metric: `dotnet.jit.compiled_methods`](#metric-dotnetjitcompiled_methods)
+  - [Metric: `dotnet.jit.compilation.time`](#metric-dotnetjitcompilationtime)
+- [Thread pool](#thread-pool)
+  - [Metric: `dotnet.thread_pool.thread.count`](#metric-dotnetthread_poolthreadcount)
+  - [Metric: `dotnet.thread_pool.work_item.count`](#metric-dotnetthread_poolwork_itemcount)
+  - [Metric: `dotnet.thread_pool.queue.length`](#metric-dotnetthread_poolqueuelength)
+- [CPU](#cpu)
+  - [Metric: `dotnet.cpu.count`](#metric-dotnetcpucount)
+  - [Metric: `dotnet.cpu.time`](#metric-dotnetcputime)
+- [General](#general)
+  - [Metric: `dotnet.monitor.lock_contentions`](#metric-dotnetmonitorlock_contentions)
+  - [Metric: `dotnet.timer.count`](#metric-dotnettimercount)
+  - [Metric: `dotnet.assembly.count`](#metric-dotnetassemblycount)
+  - [Metric: `dotnet.exceptions.count`](#metric-dotnetexceptionscount)
```

Comment:
nit: I think it should be `dotnet.exceptions` based on the previous discussion

Reply:
Sorry! I thought I'd already fixed this!